### PR TITLE
Node version documentation

### PR DESCRIPTION
**Pull request by ngraham20**
- with Node.js version
- to indicate that Volar has been moved to Vue - Official

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,15 +27,35 @@ BAR Lobby is a new WIP lobby for the RTS game [Beyond All Reason](https://www.be
...
 
 ### Requirements
 
--   [Node.js](https://nodejs.org/en/download/)
+-   [Node.js 18](https://nodejs.org/en/download/)
 
-### Project setup:
+#### Installing Node 18 on Bash
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.0/install.sh | bash
```
Comment (p2004a) [inline README.md:38]: I would just document what you need to have node 18, and for *unix* base systems we recommend nvm, and link to that docs.

I would definitely not copy nvm install instruction, pinned to specific version.

File: .gitignore [hunk 2]
```diff
@@ -15,6 +15,7 @@ pnpm-debug.log*
 
 # Editor directories and files
 .idea
+.vscode
```
Comment (p2004a) [inline .gitignore:18]: Why remove all of vscode recommended settings and so on?

File: README.md [hunk 3]
```diff
@@ -27,15 +27,35 @@ BAR Lobby is a new WIP lobby for the RTS game [Beyond All Reason](https://www.be
...
+
+nvm install 18
+
+node -v # should print `v18.20.4`
+```
+
+#### Installing Node 18 on Fish
```
Comment (p2004a) [inline README.md:45]: IMHO, fish is way too obscure to have dedicated section about it in readme, also, another reason why just pointing at nvm docs will be better.

File: .gitignore (hunk 2 above)
Comment (ngraham20) [reply .gitignore:18]: didn't realize there were specific settings tbh, I'm just used to gitignoring the IDE settings files. I'll update the PR and take that back out :+1:

File: README.md (hunk 3 above)
Comment (ngraham20) [reply README.md:45]: in my particular case, the nvm docs only *had* bash instructions, so it took a hot sec to figure it out at all. Totally fair if we don't want to include it though

**Comment by ngraham20** [comment]
@p2004a let me know if you'd prefer I do a squash commit btw. I didn't see any contributing documentation, so I'm not sure if you want the PR's commit history in here :joy:

**Review by p2004a — APPROVED** [review]
We can squash as part of accepting the PR, and by default we will be squashing ;)

**Pull request merged**